docs/help.md

```diff
@@ -1,3 +1,11 @@
 # Getting help
 
 For general assistance, write to [OnDemand@fas.harvard.edu](mailto:OnDemand@fas.harvard.edu?subject=FAS OnDemand support). Please make sure to include a link to the course for which you need help.
+
+## Providing additional information
+
+If you are able, there is additional information from OnDemand that can be helpful in diagnosing issues with the platform more quickly: your OnDemand user ID and the login node where you encountered the issue. Both of these pieces of information can be found on the main OnDemand dashboard, as illustrated below.
```

Comment:
As an idea for the future, I wonder if we could modify the [message of the day](https://github.com/fasrc/fas-ondemand-academic-config/tree/master/dashboard) that shows up on the dashboard to prepopulate this info in a `mailto:` link or just as text that the user could easily copy & paste. 

The [OOD Customization: MOTD](https://osc.github.io/ood-documentation/latest/customization.html#message-of-the-day-motd) documentation shows that `markdown_erb` is a possible format, which we could use to create a more dynamic message. I believe the `REMOTE_USER` env var holds the authenticated username, but I'm not sure about the login host info. 

The MOTD may not be the right place to do that, but just wanted to point it out as we can continue to refine things.